docs/DevGuide/ImportingData.md

```diff
@@ -0,0 +1,64 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Importing data {#dev_guide_importing}
+
+The `importers` namespace holds functionality for importing data into SpECTRE.
+We currently support loading volume data files in the same format that is
+written by the `observers`.
+
+## Importing volume data
+
+The `importers::VolumeDataReader` parallel component is responsible for loading
+volume data and distributing it to elements of one or multiple array parallel
+components. As a first step, make sure you have added the
+`importers::VolumeDataReader` to your `Metavariables::component_list`. Also make
+sure you have a `Metavariables::Phase` in which you will perform registration
+with the importer, and another for loading the data. Here's an example for such
+`Metavariables`:
+
+\snippet Test_VolumeDataReaderAlgorithm.hpp metavars
+
+To register elements of your array parallel component for receiving volume data,
+invoke the `importers::Actions::RegisterWithVolumeDataReader` action in the
+`phase_dependent_action_list`. Here's an example:
+
+\snippet Test_VolumeDataReaderAlgorithm.hpp register_action
+
+Now you're all set to load a data file. To do so, invoke the
+`importers::ThreadedActions::ReadVolumeData` action on the
```

Comment:
... action from the `execute_next_phase` function of the `ElementArray` component on the...